tests/e2e.sh

```diff
@@ -70,16 +102,21 @@ set -e
 
 kubectl version
 
+# ensure that we build docker image in minikube
+[ "$MINIKUBE_DRIVER" != "none" ] && eval $(minikube docker-env)
+
 # query kube-state-metrics image tag
 make container
 docker images -a
-ksm_image_tag=`docker images -a|grep 'quay.io/coreos/kube-state-metrics'|awk '{print $2}'|sort -u`
+ksm_image_tag=`docker images -a|grep 'quay.io/coreos/kube-state-metrics'|awk '{print $2}'|sort -u|head -n1`
```

Comment:
Why should we `head -n1`?

Reply:
In my environment (I have build multiple kms with different versions) I receive:

```bash
ksm_image_tag=`docker images -a|grep 'quay.io/coreos/kube-state-metrics'|awk '{print $2}'|sort -u`
echo $ksm_image_tag
v1.3.0
v1.2.0
```

By selecting only the first one, we choose only the image which was build with latest version.

Reply:
@dohnto There is no guarantee that the first one version is what we want. This has no problem for travis e2e environment as it is empty. This should only exist under local environment. A simple method is try to `docker rmi` all kube-state-metrics before running e2e test.

Reply:
You are right, there is no guarantee, it is a best guess.

I think that the best option is parsing the `make container` or other way of getting the $(TAG) variable from `Makefile`.

Reply:
Seems also a reasonable way.

Reply:
But maybe I can drop this change and fix it in a separate PR.

Reply:
Dropped in 84f8344.